curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-089.md

```diff
@@ -1,20 +1,34 @@
 ---
-id: 5f46e7a4750dd05b5a673920
+id: 5f46e8284aae155c83015dee
 title: Part 89
 challengeType: 0
 dashedName: part-89
 ---
 
 # --description--
 
-Now apply the `address` class to the `p` element containing the address.
+The menu looks good, but other than the coffee beans background image, it is mainly just text.
+
+Under the `Coffees` heading, add an image using the url `https://tinyurl.com/cafe-coffee-fcc`. Give the image an `alt` value of `coffee icon`.
```

Comment:
Image should be ported to our cdn.

Reply:
https://github.com/freeCodeCamp/cdn/pull/69 🙂